src/host/main.cpp

```diff
@@ -162,7 +162,16 @@ int main(int argc, char** argv)
     raft_timeout,
     "Raft timeout in milliseconds. The Raft leader sends heartbeats to its "
     "followers at regular intervals defined by this timeout. This should be "
-    "set to a significantly lower value than --raft-election-timeout-ms.",
+    "set to a significantly lower value than --consensus-election-timeout-ms.",
+    true);
+
+  size_t pbft_status_timeout = 100;
```

Comment:
Should this and `raft_timeout` be the same variable?

Reply:
I thought about this, but I would say no. `raft_timeout` is used to measure primary inactivity whereas `status_timeout` is used by all replicas to send status messages to all other replicas and has nothing to do with the primary not being responsive. Hence I think they should be separate.

Reply:
I think these could and should be combined. These have roughly the same commonality as the `election_timeout` (ie - what the _real_ timer value is, who ticks it, what a node does when it expires), we just need to be careful about the language we use to describe it. Maybe something like:

```
size_t heartbeat_timeout = 100;
app.add_option(
  "--heartbeat-timeout-ms",
  heartbeat_timeout,
  "Consensus heartbeat timeout in milliseconds. Current consensus state "
  "is shared amongst nodes at regular intervals defined by this timeout. This "
  "should be set to a significantly lower value than --election-timeout-ms."
  true);
```

I'm happy for `timeout` to be swapped for `interval` in all of these if people have strong objections to `timeout`, but I don't think there's any need to have separate arguments here on the frontend. Since the rest of this PR does a great job of _reducing_ the consensus-specific information seen by the rest of the code, it seems a shame to add a sharp distinction right on the user-facing CLI.

Reply:
If the user is setting consensus arguments, the user should be aware of what they are setting. I think it is better to explicitly name each argument and pair it with the consensus that it is referring to to avoid any developer/user confusion. Also pbft also has a null operation timer which will be exposed to the configuration in time, and that could arguably be mapped to the heartbeat timer. But since each consensus has different intentions with these timer/timeouts in mind, we should keep them separate and clearly named.

Reply:
@eddyashton the problem is that the timeouts work in different ways, and it is not possible to produce a combined description without either hiding some important details or making clear that they only apply to one variant. There are also several PBFT values that are currently hardcoded and may become configurable without an equivalent in raft.

It's also not the case that members can flip from one variant to the other without knowing anything about the differences.

I also believed that these could be combined, but after looking at the details, I am no longer convinced and I think it is in fact cleaner to keep them separate for now.